mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -119,18 +137,31 @@ CLASS ail net/minecraft/entity/Entity
 	METHOD a setRotation (FF)V
 		ARG 1 yaw
 		ARG 2 pitch
+	METHOD a updateTrackedHeadRotation (FI)V
+		ARG 1 yaw
+		ARG 2 pitch
```

Comment:
This isn't just a simple setter?

Reply:
You're right, I completely missed that detail. The integer here is consistent (called with a value of three) with the `interpolationSteps` parameter used by the other `updateTrackedX` methods.

Reply:
This is also not a simple setter. It's called by `ClientPlayNetworkHandler` in response to the `EntitySetHeadYawS2CPacket` packet. It's also a client-only method, so it's being named consistently with the other methods in its family (`updateTrackedPositionAndAngles`)